pkg/tests/testinfra/testinfra.go

```diff
@@ -487,6 +483,29 @@ func CreateGrafDir(t *testing.T, opts GrafanaOpts) (string, string) {
 	return tmpDir, cfgPath
 }
 
+func createGcomApiNToken(t *testing.T, opts GrafanaOpts, cfg *ini.File) {
+	getOrCreateSection := func(name string) (*ini.Section, error) {
+		section, err := cfg.GetSection(name)
+		if err != nil {
+			return cfg.NewSection(name)
+		}
+		return section, err
+	}
+
+	if opts.GrafanaComAPIURL != "" {
+		grafanaComSection, err := getOrCreateSection("grafana_com")
+		require.NoError(t, err)
+		_, err = grafanaComSection.NewKey("api_url", opts.GrafanaComAPIURL)
+		require.NoError(t, err)
+	}
+	if opts.GrafanaComSSOAPIToken != "" {
+		grafanaComSection, err := getOrCreateSection("grafana_com")
+		require.NoError(t, err)
+		_, err = grafanaComSection.NewKey("sso_api_token", opts.GrafanaComSSOAPIToken)
+		require.NoError(t, err)
+	}
+}
```

Comment:
Suggestion: If the `gocyclo` linter is annoying you can also use `//nolint:gocyclo` to disable it in `CreateGrafDir`. The function is only used for testing, so it's not a big deal in this case